src/HomeWork/hw5/Main.java

```diff
@@ -0,0 +1,32 @@
+package HomeWork.hw5;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("===== MyHashMap =====");
+        MyHashMap<String, Integer> myHashMap = new MyHashMap<>();
+        myHashMap.put("Sviatik", 21);
```

Comment:
Verify case when put entries with same key and case with entries with same hashcode

Reply:
there is one case with entries with same key:
`        myHashMap.put("Sviatik", 21);
        myHashMap.put("Misha", 25);
        myHashMap.put("Vova", 25);

        System.out.println("put(Misha, 10000): ");
        myHashMap.put("Misha", 10000);
`

Reply:
there is one case with entries with same key
```

        myHashMap.put("Sviatik", 21);
        myHashMap.put("Misha", 25);
        myHashMap.put("Vova", 25);

        System.out.println("put(Misha, 10000): ");
        myHashMap.put("Misha", 10000);
```

Reply:
also i changed getHashCode() method, it returns always 2 and all the elements put in the same "bucket"